src/services/sync/repo/user.repo.js

```diff
@@ -97,8 +97,9 @@ const updateAccount = async (userId, account) =>
 
 const updateUserAndAccount = async (userId, changedUser, changedAccount) => {
 	await checkUpdate(changedUser.email, userId);
-	changedUser.roles = await resolveUserRoles(changedUser.roles);
-
+	if('roles' in changedUser){
```

Comment:
What about this way?
```suggestion
	if(changedUser.roles){
```